File: sdk/autoDataSource-sdk/go/test_datasource_list.go
package main

import (
	"fmt"

	"./"
)

func main() {
	// 初始化客户端
	baseURL := "http://localhost:8080/autoDataSource"
	client := NewClient(baseURL)

	// 获取数据源列表
	dataSourcesResp, err := client.GetDataSources()
	if err != nil {
		fmt.Printf("获取数据源列表失败: %v\n", err)
	} else {
		fmt.Printf("数据源列表响应: %+v\n", dataSourcesResp)
		fmt.Println("成功获取数据源列表！")
	}
}

File: sdk/autoDataSource-sdk/java/TestDataSourceList.java
import com.example.autodatasource.sdk.AutoDataSourceClient;
import java.util.Map;

public class TestDataSourceList {
    public static void main(String[] args) {
        // 初始化客户端
        String baseUrl = "http://localhost:8080/autoDataSource";
        AutoDataSourceClient client = new AutoDataSourceClient(baseUrl);

        try {
            // 获取数据源列表
            Map<String, Object> dataSourcesResponse = client.getDataSources();
            System.out.println("数据源列表响应: " + dataSourcesResponse);
            System.out.println("成功获取数据源列表！");
        } catch (Exception e) {
            System.err.println("获取数据源列表失败: " + e.getMessage());
            e.printStackTrace();
        }
    }
}